crates/ruma-client-api/src/media/get_content.rs

```diff
@@ -97,8 +108,15 @@ pub mod v3 {
 
     impl Response {
         /// Creates a new `Response` with the given file contents.
+        ///
+        /// The `Cross-Origin Resource Policy` defaults to `cross-origin`.
```

Comment:
```suggestion
        /// The `Cross-Origin-Resource-Policy` defaults to `cross-origin`.
```

… or maybe the same as before but w/o a code block.